zerver/lib/home.py

```diff
@@ -226,5 +229,6 @@ def build_page_params_for_home_page_load(
         page_params["enable_desktop_notifications"] = False
 
     page_params["translation_data"] = get_language_translation_data(request_language)
+    page_params["giphy_api_key"] = get_giphy_api_key()
```

Comment:
These should probably go in `zerver/lib/events.py` -- at least if we imagine we'll eventually want the mobile apps to be able to integrate this as well.

Reply:
Correct.